src/Benchmarks/Startup.cs

```diff
@@ -54,8 +55,15 @@ public void ConfigureServices(IServiceCollection services)
             
             if (Scenarios.Any("Raw") || Scenarios.Any("Dapper"))
             {
-                // TODO: Add support for plugging in different DbProviderFactory implementations via configuration
-                services.AddSingleton<DbProviderFactory>(SqlClientFactory.Instance);
+                var database = Configuration["database"] ?? "sqlserver";
```

Comment:
Can this be changed to use the strongly-typed AppSettings, rather than the string from config?